cli/src/opts/cast.rs

```diff
@@ -254,6 +254,8 @@ pub enum Subcommands {
         confirmations: usize,
         #[clap(long = "json", short = 'j')]
         to_json: bool,
+        #[clap(long = "resend", help = "reuse nonce")]
+        resend: bool,
```

Comment:
this should be conflicts_with = nonce, right?
can we also be a bit more explicit in the help message?